Headers/DebugServer2/Architecture/ARM64/CPUState.h

```diff
@@ -38,10 +38,10 @@ typedef ds2::Architecture::ARM::CPUState CPUState32;
 //
 struct CPUState64 {
   union {
-    uint64_t regs[32 + 1 + 1];
+    uint64_t regs[31 + 1 + 1 + 1];
```

Comment:
That's just a semi-useful way of keeping track of where these registers come from. There are 31 GP registers (x0...x30) and sp, pc, and cpsr. This change doesn't have any actual impact on how the code runs.